src/app/features/schedule/schedule.component.scss

```diff
@@ -0,0 +1,17 @@
+table {
```

Comment:
If you want using `tag` to stylesheet, please define it on `_base.scss` or `_common.scss`.